### ISSUE TITLE
Insomnia has become unresponsive

### ISSUE THREAD

**Issue by purwowd** [getinsomnia/insomnia#5087]
### Expected Behavior

The expectation is not an issue when opening the app

### Actual Behavior

The window blanks and shows an error popup: "Insomnia has become unresponsive. Do you want to reload?"

<img width="266" alt="image" src="https://user-images.githubusercontent.com/17087219/185019947-5bb4983d-77b9-4017-9c9f-f1d2a4b27078.png">

### Is there an existing issue for this?

- [X] I have searched the [issue tracker](https://www.github.com/Kong/insomnia/issues) for this problem.

### Insomnia Version

2022.5.0

### What operating system are you using?

macOS

### Operating System Version

macOS Monterey 12.5

### Installation method

download from insomnia.rest

**Comment by kreosus** [comment]
This sounds like it could be a bug.

Hi @purwowd  Please reply with the following information:

- What version of Insomnia are you using?
- What operating system and version is this on?
- How did you install Insomnia?

Try moving the application data, and see if that affects anything. Moving the folder instead of deleting it means you can restore it later.

App Data on macOS:  /Users/<your user>/Library/Application Support/Insomnia
App Data for Windows:  %APPDATA%/Insomnia

Cheers,

**Comment by purwowd** [comment]
Hi @kreosus, thanks for your suggestion. I tried to move the folder and it works, but all of the data HTTP requests in my folder have also been deleted.

And then I found the issue is the timeout response for parsing CSV from my HTTP request, and insomnia can't handle it.

Finally, my solution is to delete the responses folder on App Data. It works and all of the data HTTP requests in my folder still exist.

Location folder: /Users//Library/Application Support/Insomnia/responses/

Cheers...
